package.json

```diff
@@ -27,6 +28,7 @@
     "@reduxjs/toolkit": "^1.8.6",
     "@rematch/core": "^2.2.0",
     "core-js": "^3.6.5",
+    "cors": "^2.8.5",
```

Comment:
Is this package a peer dependency?  I don't see it getting used

Reply:
Mentioned in, 
https://github.com/the-ai-team/razor/pull/53/files#r1119848113